### PR TITLE
update release workflow

### PR DESCRIPTION
**Pull request by knaaptime**

### CODE REVIEW

File: .github/workflows/upload_package.yml [hunk 1]
```diff
@@ -1,53 +1,45 @@
...
         python-version: '3.x'
+    
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install --upgrade build twine
-        python -m build
-        twine check --strict dist/*
+        pip install setuptools wheel twine jupyter urllib3 pandas pyyaml
+        python setup.py sdist bdist_wheel
```
Comment (knaaptime) [inline .github/workflows/upload_package.yml:29]: @jGaboardi does this line with setup.py still work with pyproject.toml? I thought you had to use `pip install .` ?

Comment (jGaboardi) [reply .github/workflows/upload_package.yml:29]: See [here](https://github.com/pysal/spaghetti/blob/93cd5346eed24db92c65bc07eef74d6d78487903/.github/workflows/release_and_publish.yml#L34) in `spaghetti`'s release workflow.

Comment (knaaptime) [reply .github/workflows/upload_package.yml:29]: ah ok thanks. I snagged the last one from spopt

Comment (jGaboardi) [reply .github/workflows/upload_package.yml]: Have not had enough time recently to sync everything 😰

**Pull request merged**